scripts/generate_proto_stubs.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # Some of the proto .pyi stubs in stubs/protobuf/
 # are autogenerated using the mypy-protobuf project on the
 # latest `.proto` files shipped with protoc.
-#
+
+set -ex
```

Comment:
We should add `-o pipefail`.
```suggestion
set -ex -o pipefail
```